GG/test/acceptance/runner/Application.cpp

```diff
@@ -181,19 +181,39 @@ void MinimalGGApp::Render() {
 // application-dependent.  Note that this method is called before Render() is
 // ever called.
 void MinimalGGApp::GLInit() {
-    double ratio = Value(AppWidth() * 1.0) / Value(AppHeight());
-
     glEnable(GL_BLEND);
     glEnable(GL_CULL_FACE);
     glEnable(GL_DEPTH_TEST);
     glClearColor(0, 0, 0, 0);
     glViewport(0, 0, Value(AppWidth()), Value(AppHeight()));
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
-    gluPerspective(50.0, ratio, 1.0, 10.0);
-    gluLookAt(0.0, 0.0, 5.0,
-              0.0, 0.0, 0.0,
-              0.0, 1.0, 0.0);
+
+    // calculate perspective with vertical FOV of 50°. 1:1 application
+    // window ratio, near plane of 1.0 and far plane of 10.0
+    float ratio = Value(AppWidth() * 1.0) / Value(AppHeight());
+    float radians = 50.0f * M_PI / 90.0f;
+    float cotangent = std::cos(radians) / std::sin(radians);
+
+    float projection[4][4] = { 0.0f };
+    projection[0][0] = cotangent / ratio;
+    projection[1][1] = cotangent;
+    projection[2][2] = -11.0f / 9.0f;
```

Comment:
Lot of unexplained "magic" numbers here...  a comment indicating where they came from might be good. Maybe also rephrase "calculate" to "set up" above, since not much calculation is actually happening...

Reply:
That said, I don't consider this test code particularly important, so if you'd prefer to not bother commenting it more, it's fine with me.

Reply:
Replaced the non-magic magic numbers with variable within the matrix set up.